documentation/documentation.css

```diff
@@ -138,7 +138,9 @@
 #button-container,
 #cards-container,
 #form-container,
-#responsive-image-ctn {
+#responsive-image-ctn ,
```

Comment:
Similarly here also what does **-ctn** mean?

Reply:
removed all the id's related CSS !!

Reply:
ok good 😊